scene/3d/voxel_gi.cpp

```diff
@@ -338,7 +347,7 @@ void VoxelGI::_find_meshes(Node *p_at_node, List<PlotMesh> &plot_meshes) {
 
 	Node3D *s = Object::cast_to<Node3D>(p_at_node);
 	if (s) {
-		if (s->is_visible_in_tree()) {
+		if (is_node_voxel_bakeable(s)) {
 			Array meshes = p_at_node->call("get_meshes");
```

Comment:
To clarify my earlier comment. I would add:
```suggestion
Array meshes;

MultiMeshInstance3D *mmi = Object::cast_to<MultiMeshInstance3D>(p_at_node);
if (mmi ) {
    // Create meshes from MultiMesh properties
} else {
meshes = p_at_node->call("get_meshes");
}
```